test/test_autograd.py

```diff
@@ -2094,6 +2094,16 @@ def test_cat_empty_legacy(self):
                               False, f_args_variable, f_args_tensor)
         self.assertTrue(gradcheck(lambda a, b: torch.cat((a, b)), f_args_variable, eps=1e-6, atol=PRECISION))
 
+    def test_cdist(self):
```

Comment:
can you move this above or below all the `cat` tests?

Reply:
Fixed